src/py/aspen/covidhub_import/implementation.py

```diff
@@ -97,24 +126,85 @@ def import_project(
     with session_scope(interface) as session:
         project = (
             covidhub_session.query(Project)
+            .options(
+                joinedload(
+                    Project.covidtracker_group, covidtracker.GroupToProjects.group
+                )
+            )
             .filter(Project.rr_project_id == rr_project_id)
             .one()
         )
 
-        group = (
-            session.query(Group)
-            .filter(Group.name == project.originating_lab)
-            .one_or_none()
-        )
-        if group is None:
-            # copy the project info into the "group"
-            group = Group(
-                name=project.originating_lab,
-                address=project.originating_address,
+        group, admin_group, cdph_group = (
+            get_or_make_group(
+                session,
+                project.originating_lab,
+                project.originating_address,
+                # FIXME: this needs to be updated.
+                email="fakeemail@dph.gov",
```

Comment:
i thought that this field had to be unique, or did we disable that until we have a workaround?

Reply:
will this break if we are moving over multiple groups (besides admin and cdph)?

Reply:
Yeah, this will break with multiple groups.